pandas/io/parsers.py

```diff
@@ -2273,10 +2278,11 @@ def __init__(self, f, **kwds):
             raise ValueError('Only length-1 decimal markers supported')
 
         if self.thousands is None:
-            self.nonnum = re.compile('[^-^0-9^%s]+' % self.decimal)
+            self.nonnum = re.compile(
+                r'[^-^0-9^{decimal}]+'.format(decimal=self.decimal))
```

Comment:
Hmm does converting this to a raw string when it wasn't previously affect the behavior at all? Might also be the root cause of your issue, though we technically drop Py2 support with v0.25 anyway

Reply:
Temporarily, I want to try to put that code back for clearing the problem from ci.
And then I'll find out how can I solve the problem.

Reply:
This change has no problem :)